### DISCUSSION TITLE
Fabric gotchas link is broken

### DISCUSSION THREAD

**Discussion by leszekhanusz** [fabricjs/fabric.js#8234] [General]
It seems the Gotcha's [https://github.com/fabricjs/fabric.js/wiki/Fabric-gotchas](https://github.com/fabricjs/fabric.js/wiki/Fabric-gotchas) link is broken.

**Comment by ShaMan123** [comment]
Sorry about that
#8189  to fix links is in queue

**Comment by ShaMan123** [comment]
I opened #8236
